lib/traject/macros/met_csv.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting values from CSV rows
+  module MetCsv
+    # @param header_or_index [String] the field header or index to accumulate
+
+    def generate_creator
+      lambda do |row, accumulator, _context|
+        display_pattern = row['Artist Display Name'] unless row['Artist Display Name'].empty?
+        display_pattern = "#{display_pattern}, #{row['Artist Suffix']}" unless row['Artist Suffix'].empty?
```

Comment:
What if display_pattern was nil here? Then it would begin with a comma and space. That seems odd